### ISSUE TITLE
Issue with combined DOE/DEER create_typical_building_from_model measure

### ISSUE THREAD

**Issue by DavidGoldwasser** [NREL/openstudio-extension-gem#55]
The DEER and DOE unique versions of create typical and of create bar do not ask fro climate zones, as they expect them to already be in the model (often added by ChangeBuildingLocation measure. There appears to be an issue with how the climate_zone argument in the combined version is mapped to the openstudio-standards lookup. It works fine when "Lookup From Model" is used.
https://github.com/NREL/openstudio-extension-gem/blob/develop/lib/openstudio/extension/core/os_lib_model_generation.rb#L2649-L2655

@asparke2 I can certainly fix this, but the "climate_zone" argument was only in there in the combined version because comstock and or LA100 needed it. If that is not the case any more then I can just remove that argument, but if you think you still need it in the future then I'll keep it in and fix it.

**Comment by DavidGoldwasser** [comment]
As a note the combined DOE/DEER measures (create_bar and create_typical) aren't for general use since there are so may bad climate zone/ building type/ template combinations that can be chosen in a GUI that doesn't constrain the choices.

**Comment by asparke2** [comment]
I think we should keep it in there and fix for now.  The current ComStock version is here:  https://github.com/NREL/ComStock/blob/develop/resources/measures/create_typical_building_from_model/measure.rb, and the current LA100 (DEER) version is here:  https://github.com/NREL/la100-comstock/blob/develop/resources/measures/create_typical_building_from_model/measure.rb.  We haven't merged these together, but they each now work independently.  There were some recently climate zone related changed to the ComStock one and handling climate zones 1, 7, and 8, where they sometimes have a letter (A) and sometimes don't, depending on where you look.

**Comment by DavidGoldwasser** [comment]
So there were a combination of things that happened to cause this error.
- The combined create_typical measure wash chosen (vs. doe or deer which don't have climate_zone)
- a DEER building type was used
- Change Building Location was upstream in the workflow
-  "Use upstream measures" was checked

I wrote a test with Sammy's inputs and it passed
https://github.com/NREL/openstudio-model-articulation-gem/commit/96c7ef3994d2f933de5b567990c1cafe35502964

I then noticed the combined create_bar and create_typical use a `get_cliimate_zones` method that pulls from the extension gem. There are similar `get_building_types` and `get_templates` methods. which returns. `CEC T24-CEC5`
https://github.com/NREL/openstudio-extension-gem/blob/cbcbe47008d42e7c72dd0f897cf59351bc03e17a/lib/openstudio/extension/core/os_lib_model_generation.rb#L270

ChangeBuildingLocation uses its own hard coded list of valid inputs and does not yet use the methods in the extension gem. it has value of `T24-CEC5`
https://github.com/NREL/openstudio-common-measures-gem/blob/develop/lib/measures/ChangeBuildingLocation/measure.rb#L91

I think the proper fix is to have change building location, and any measures that request climate zones use the methods in the extension gem; however I don't feel a rush to do that now, I think it can wait until after the release. So this will end up bing a fix in model articulation measure and not in the extension-gem, however there is a question of if we want the longer name that has CEC in it twice. Not a big deal either way for me. Note, that the ASHRAE climate zones are in sync and don't exhibit this issue.
